resources/templates/boardlist.json

```diff
@@ -38,14 +31,6 @@
                 "architecture": "esp32"
             },
             "projectHostType": "workspace"
-        },
-        {
-            "name": "Raspberry Pi",
-            "id": "raspberrypi",
-            "detailInfo": "Raspberry Pi",
-            "helpUrl": "https://www.raspberrypi.org/",
-            "exampleUrl": "https://raw.githubusercontent.com/VSChina/azureiotdevkit_tools/gallery/workbench-example-raspi.json",

```

Comment:
You could just remove the example Url here.
No need to remove Rpi as the supported device.

Reply:
Fixed.